hub/powertoys/run.md

```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+
 
 ## Settings
 
-The following general options are available for PowerToys Run in the PowerToys settings menu.
+The following general options are available on the PowerToys Run settings page.
 
-| Settings | Description |
+| Setting | Description |
 | :--- | :--- |
-| Activation shortcut | Define the keyboard shortcut to open/hide PowerToys Run |
-| Use centralized keyboard hook | You can try this setting if there are issues with the shortcut 
+| Activation shortcut | Define the keyboard shortcut to show/hide PowerToys Run |
+| Use centralized keyboard hook | Try this setting if there are issues with the keyboard shortcut
 | Ignore shortcuts in Fullscreen mode | When in full-screen (F11), PowerToys Run won't be engaged with the shortcut |
 | Maximum number of results | Maximum number of results shown without scrolling |
 | Clear the previous query on launch | When launched, previous searches will not be highlighted |
-| Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
+| Preferred display position | If multiple displays are in use, PowerToys Run can be launched on:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
 | App theme | Change the color theme used by PowerToys Run |
 
 
-## Usage
-
-### Keyboard shortcuts
+### Plugin manager
 
-| Shortcuts | Action |
-| :--- | :--- |
-|<kbd>Alt</kbd>+<kbd>Space</kbd> (default) | Open or hide PowerToys Run |
-|<kbd>Esc</kbd> | Hide PowerToys Run |
-|<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Enter</kbd> | Open the selected application as administrator (only applicable to applications) |
-|<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>U</kbd> | Open the selected application as different user (only applicable to applications) |
-|<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd> | Open containing folder in File Explorer (only applicable to applications and files) |
-|<kbd>Ctrl</kbd>+<kbd>C</kbd> | Copy path location (only applicable to folders and files) |
-|<kbd>Tab</kbd> | Navigate through the search result and context menu buttons |
+PowerToys Run uses a plugin system to provide different types of results. The settings page includes a plugin manager that allows you to enable/disable the various available plugins. By selecting and expanding the sections, you can customize the direct activation commands used by each plugin. In addition, you can select whether a plugin appears in global results and set additional plugin options where available.
 
+![PowerToys Run Plugin Manager](../images/pt-run-plugin-manager.png)
 
-### Direct activation commands
+#### Direct activation commands
 
-These default direct activation commands will force PowerToys Run into only targeted plugins.
+A number of the plugins can be activated with a direct activation command so that PowerToys Run will only use the targeted plugin.
```

Comment:
This is incorrect. You can have an activation command for all plugins. But we don't have one for each by default.

Should be something like: "The plugins... The following table shows the direct activation commands assigned by default."